code/__DEFINES/traits/declarations.dm

```diff
@@ -818,6 +818,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /* #define TRAIT_HEAVY_DRINKER "heavy_drinker" */
 #define TRAIT_AGEUSIA "ageusia"
 #define TRAIT_HEAVY_SLEEPER "heavy_sleeper"
+#define TRAIT_LIGHT_SLEEPER "light_sleeper" //monke edit
```

Comment:
just add these to `code/__DEFINES/traits/monkestation/declarations.dm`

Reply:
ah you right